src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.component.html

```diff
@@ -0,0 +1,176 @@
+<div class="d-flex flex-wrap gap-2 fs-xx-small align-items-center" [style.width]="getPostingType() === 'post' ? 'max-content' : ''">
+    @if (getPostingType() === 'post') {
+        @if (hoverBar() && sortedAnswerPosts()?.length === 0) {
+            <div>
+                <button class="reaction-button clickable reply-btn" (click)="isCommunicationPage() ? openThread.emit() : openAnswerView()">
+                    <fa-icon class="fa-xs align-self-center" [icon]="faArrowRight" />
+                    <span jhiTranslate="artemisApp.conversationsLayout.threadSideBar.reply" class="emoji-count px-2"></span>
+                </button>
+            </div>
+        }
+        @if (!isCommunicationPage()) {
+            @if (sortedAnswerPosts()?.length) {
+                @if (showAnswers()) {
```

Comment:
I'm not sure how easy it is to refactor/ how much it would improve readability, but such nested `if`s are generally an anti-pattern https://thewayofpython.com/2023/05/01/hidden-and-the-anti-pattern/
Maybe you could try to disentagle the nesting a bit here